lib/logstash/inputs/udp.rb

```diff
@@ -50,6 +51,7 @@ def run(output_queue)
       udp_listener(output_queue)
     rescue => e
       @logger.warn("UDP listener died", :exception => e, :backtrace => e.backtrace)
+      @metric_errors.increment(:listener_exception)
```

Comment:
I agree on having uniform patterns for namespaces, I created https://github.com/elastic/logstash/issues/5716 to centalize discussions and reach consensus 👍